h2o-r/h2o-package/R/models.R

```diff
@@ -3064,5 +3064,80 @@ h2o.deepfeatures <- function(object, data, layer) {
   h2o.getFrame(dest_key)
 }
 
+#'
+#' The H2OTree class.
+#'
+#' This class represents a model of a Tree built by one of H2O's algorithms (GBM, Random Forest).
+#' @slot left_children An \code{integer} vector with left child nodes of tree's nodes
+#' @slot right_children An \code{integer} vector with right child nodes of tree's nodes
+#' @slot descriptions A \code{character} vector with descriptions for each node to be found in the tree. Contains split threshold if the split is based on numerical column.
+#'                    For cactegorical splits, it contains list of categorical levels for transition from the parent node.
+#' @slot model_id A \code{character} string with the name of the model this tree has been generated in.
+#' @slot tree_number An \code{integer} representing the order in which the tree has been built in the model.
```

Comment:
from a user pov, not a big fan of the word `number` in general as it is not very informative usually.
tree_index maybe? 
this also applies to the java API

Reply:
This is a valid point. We'll consider it in the next updates.